regions/core/pixcoord.py

```diff
@@ -90,13 +94,7 @@ def _validate(val, name, expected='any'):
 
     @property
     def isscalar(self):
-        """Is this pixcoord a scalar? (a bool property)"""
-        # TODO: what's the best solution to implement this?
-        # Maybe we should sub-class ShapedLikeNDArray?
-        # See https://github.com/astropy/regions/issues/108
-        # This is a temp solution that matches the bahaviour of SkyCoord
-        return np.asanyarray(self.x).shape == ()
-        # return np.isscalar(self.x) and np.isscalar(self.y)
+        return np.isscalar(self.x)
```

Comment:
Do we still match the behaviour of SkyCoord? Do we have any tests for that? If yes, OK.

Reply:
I think so. I have added some tests for this now.

Reply:
The tests look good. Thanks!